src/NzbDrone.Core/Indexers/Definitions/Anilibria.cs

```diff
@@ -0,0 +1,341 @@
+            }
+
+            title += " " + tl.Season.Year;
+            title += " [" + tr.Quality.String + "]";
+            if (!string.IsNullOrWhiteSpace(tr.Series.String))
+            {
+                title += " - E" + tr.Series.String;
+            }
+
+            return title;
+        }
+
+        public IList<ReleaseInfo> ParseResponse(IndexerResponse indexerResponse)
+        {
+            var torrentInfos = new List<ReleaseInfo>();
+            var queryResponseItems = JsonConvert.DeserializeObject<List<AnilibriaTitle>>(indexerResponse.Content);
+
+            foreach (var tl in queryResponseItems)
+            {
+                foreach (var tr in tl.Torrents.List)
+                {
+                    var torrentInfo = new TorrentInfo
+                    {
+                        Title = composeTitle(tl, tr),
+                        InfoUrl = string.Format("{0}/release/{1}.html", _settings.BaseUrl.TrimEnd('/'), tl.Code),
+                        DownloadVolumeFactor = 0,
+                        UploadVolumeFactor = 1,
+                        Categories = new[]
+                        {
+                            NewznabStandardCategory.TVAnime
+                        },
+                        Seeders = tr.Seeders,
+                        Peers = tr.Leechers + tr.Seeders,
+                        Grabs = tr.Downloads,
+
+                        // API provides timestamp in UTC+3 timezone, so we need to substract 3 hours
+                        PublishDate = DateTimeUtil.UnixTimestampToDateTime(tr.UploadedTimestamp).AddHours(-3),
+                        Guid = _settings.BaseUrl + tr.Url,
+                        DownloadUrl = _settings.BaseUrl + tr.Url,
+                        Size = tr.TotalSize,
+                        Resolution = tr.Quality.Resolution,
+                        Codec = tr.Quality.Encoder
+                    };
+                    torrentInfos.Add(torrentInfo);
+                }
+            }
+
+            return torrentInfos.ToArray();
+        }
+
+        public Action<IDictionary<string, string>, DateTime?> CookiesUpdater { get; set; }
+    }
+
+    public class AnilibriaSettingsValidator : AbstractValidator<AnilibriaSettings>
+    {
+        public AnilibriaSettingsValidator()
+        {
+            RuleFor(c => c.ApiUrl).NotEmpty();
+        }
+    }
+
+    public class AnilibriaSettings : IIndexerSettings
+    {
+        private static readonly AnilibriaSettingsValidator Validator = new AnilibriaSettingsValidator();
+
+        public AnilibriaSettings()
+        {
+            ApiUrl = "https://api.anilibria.tv/";
+        }
+
+        [FieldDefinition(1, Label = "Base Url", Type = FieldType.Select, SelectOptionsProviderAction = "getUrls", HelpText = "Select which baseurl Prowlarr will use for requests to the site")]
+        public string BaseUrl { get; set; }
+
+        [FieldDefinition(2, Label = "Api Url", HelpText = "URL of api")]
```

Comment:
Why have a separate setting, does this ever change?

Reply:
Also, no API key?

If there is one to be input, it should be formatted in as follows, editing helptext as needed `Label = "API Key", HelpText = "API Key from the Site (Found in My Security => API Key)", Privacy = PrivacyLevel.ApiKey)]`

Reply:
About separate settings. As you can see main api is a subdomain of main site. And api responses contents relative links (without specifying) domain on main site.

Is it better to keep only one setting (Base URL) and form api url by adding `api.` subdomain to Base URL?

About API keys - this tracker is public, there's no API keys on site, but it's possible to provide seesionID (of login/password auth) to requests. But based on API documentation (https://github.com/anilibria/docs/blob/master/api_v2.md#%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D0%B5%D0%BB%D1%8C%D1%81%D0%BA%D0%B8%D0%B5-%D0%BC%D0%B5%D1%82%D0%BE%D0%B4%D1%8B-%D1%82%D1%80%D0%B5%D0%B1%D1%83%D1%8E%D1%89%D0%B8%D0%B5-%D0%B0%D0%B2%D1%82%D0%BE%D1%80%D0%B8%D0%B7%D0%B0%D1%86%D0%B8%D0%B8) it is only needed to operate favourite lists on site.

Reply:
> Is it better to keep only one setting (Base URL) and form api url by adding api. subdomain to Base URL?

id say so to avoid confusion and possible mismatchs

Reply:
Replaced API URL with getting api url from baseURL via regexp